examples/pipelines/asl_nistats_processing.py

```diff
@@ -0,0 +1,170 @@
+"""ASL processing with nistats
+"""
+
+import os
+import os.path as op
+import numpy as np
+import nibabel as nb
+import glob
+import pandas as pd
+import matplotlib.pyplot as plt
+import nilearn
+import scipy
+
+from collections import OrderedDict
+from nistats import experimental_paradigm, design_matrix, glm
+from nistats.glm import FirstLevelGLM
+from nistats.thresholding import map_threshold
+from nilearn.input_data import NiftiMasker
+from nilearn.plotting import plot_stat_map
+
+from pyhrf.retreat.glm_tools import make_design_matrix_asl
+from pypreprocess.reporting.glm_reporter import group_one_sample_t_test
+from pypreprocess.external.nistats.glm import FMRILinearModel
```

Comment:
I don't want to rely on `pypreprocess` either, but the nistats first level model is not merged yet, so let's use `pypreprocess` temporary.
Could you please add `pypreprocess` version requirement to the file `requirements.txt` ?